### ISSUE TITLE
Need admin user summary view

### ISSUE THREAD

**Issue by drshellen** [drshellen/DiscordRPBot#11]
Need to create a command to view all users/NPCs and some basic stats.

**Comment by cjwomack** [comment]
What's a NPC?

**Comment by cjwomack** [comment]
non-player character

**Comment by drshellen** [comment]
Yes - if you view the current character commands, characters are only created by the actual server member.  This doesn't allow the server admin/DM to create a NPC.  
It may be a good idea to allow the DM to create a character for the server members as well.  
@cjwomack
